GOVERNANCE.md

```diff
@@ -1,73 +1,95 @@
-# Project Governance
+# The Organization
 
-_NOTE_: This document is not final. See [RFC: Technical Steering Committee style governance #1](https://github.com/lexicon-community/governance/discussions/1)
+The mission of the Lexicon Community is to create an effective, active blueprint for collective ownership of Lexicons in the ATMosphere and build a stable and thoughtful lexicon that can be used by application, system, and SDK developers.
```

Comment:
Would be nice to mention "lexicon-related tooling" in the mission also, but not a blocker.

Reply:
```suggestion
The mission of the Lexicon Community is to create an effective, active blueprint for collective ownership of Lexicons in the ATMosphere and build a stable and thoughtful lexicon that can be used by application, system, and lexicon-related tooling developers.
```

Like this?